tests/common/request.py

```diff
@@ -41,10 +43,29 @@ def get_transaction_by_id(transaction_id: str):
     return raw_response.json()
 
 
-def post_request_and_wait_for_finalization(
-    payload: dict, interval: int = 10, retries: int = 15
+def call_contract_method(
```

Comment:
New backend integration tests helpers to use the send_raw_transaction RPC method